.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I guess it's better to not use custom file extensions when it is possible, it's better to use regular/common file extensions.
For example for `.npmrc`, we can't have another file extensions (as far as I know).

But for `prettier`, we can, for example, we could use `.prettierrc.json` instead, so for IDE that don't know `prettier`, we can still have auto completion and syntax highlighting etc, as it is the case in `github` (syntax highlighting doesn't work on `.prettierc` file).

Reply:
I've worked on projects with all these type of files and never had issue with any of them. Any decent IDE can handle them accordingly. So I guess it's a matter of preference :-) I think both prittier and eslint support no extention, json and js. I like js to keep syntax consistent with code, but it's easily changable.

Reply:
I don't think it's a matter of preference, and I don't see any advantages to use `.prettierc`.
> I like js to keep syntax consistent with code, but it's easily changeable.

Why if you like, js extensions, you didn't use it for the `prettier` config ?
You used it for `eslint` but not for `prettier`, it makes the code inconsistent.

Reply:
Oh god, sorry, I haven't been paying too much attention. Fixed in new commit